tests/test_plotting.py

```diff
@@ -778,14 +778,52 @@ def test_plot_boozer_modes():
     """Test plotting boozer spectrum."""
     eq = get("WISTELL-A")
     fig, ax, data = plot_boozer_modes(
-        eq, M_booz=eq.M, N_booz=eq.N, num_modes=7, return_data=True
+        eq, M_booz=eq.M, N_booz=eq.N, num_modes=7, return_data=True, norm=True
```

Comment:
Why this change? The other two tests you added also use `norm=True`, so it could be nice to keep this as `norm=False` for more coverage.

Reply:
Mainly for visual consistency. If they're all normed all the plots line up so its easier to tell that everything is doing what it should be

Reply:
It shouldn't effect the coverage at all, the way its coded `norm=True` exercises more code